docs/features/data_in_dynamodb.md

```diff
@@ -13,71 +13,8 @@ hide_title: true
 
 All configurations and system-generated metadata is stored in DynamoDB tables except the logs. System logs are stored in the AWS CloudWatch service.
 
-`@cumulus/api` uses the following DynamoDB tables:
-
-- **Collections:** collection records
-- **Providers:** provider records
-- **Rules:** rules for managing and running workflows
-- **Executions:** workflow executions (step function executions)
-- **Granules:** granules processed by the Cumulus instance
-- **PDRs:** PDRs processed in Cumulus
-
 Amazon DynamoDB stores three geographically distributed replicas of each table to enable high availability and data durability. Amazon DynamoDB runs exclusively on solid-state drives (SSDs). SSDs help AWS achieve the design goals of predictable low-latency response times for storing and accessing data at any scale.
 
-## Backup and Restore with AWS
```

Comment:
The header on line 9 should be removed as well if this selection is deleted

Reply:
Good catch. Thank you!